src/features/transaction-wizard/mappers/as-description-list-items.tsx

```diff
@@ -518,7 +523,7 @@ const asResourcesItem = (accounts?: string[], assets?: bigint[], apps?: bigint[]
                 <ol className="pl-4">
                   {assets.map((assetId, index, array) => (
                     <li key={index} className="truncate">
-                      <AssetIdLink assetId={Number(assetId)} className="inline text-primary underline">
+                      <AssetIdLink assetId={assetId} className="inline text-primary underline">
                         {Number(assetId)}
```

Comment:
```suggestion
                        {assetId.toString()}
```
Potentially this is better handling now?
Probably worth doing a search for all the places.